File: Statement Level Control/GuardClausees.py
def process_order(order):
    if not order:
        return "No order provided"
    if not order.get("items"):
        return "Order has no items"
    if order.get("total", 0) < 0:
        return "Invalid total"
    return f"Processing order of {order['total']}"


print(process_order(None))
print(process_order({"items": [], "total": 0}))
print(process_order({"items": ["rice"], "total": 50}))

File: Statement Level Control/GuardClausees.js
function processOrder(order) {
  if (!order) return "No order provided";          // Guard 1
  if (!order.items || order.items.length === 0)
    return "Order has no items";                   // Guard 2
  if (order.total < 0) return "Invalid total";     // Guard 3

  // Main logic only reached if all guards pass
  return `Processing order of ₱${order.total}`;
}

console.log(processOrder(null));                        // "No order provided"
console.log(processOrder({ items: [], total: 0 }));    // "Order has no items"
console.log(processOrder({ items: ["rice"], total: 50 })); // "Processing order of ₱50"